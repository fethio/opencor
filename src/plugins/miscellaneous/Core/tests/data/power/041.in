<math xmlns="http://www.w3.org/1998/Math/MathML">
    <apply>
        <eq/>
        <ci>a</ci>
        <apply>
            <minus/>
            <apply>
                <power/>
                <apply>
                    <root/>
                    <degree>
                        <ci>c</ci>
                    </degree>
                    <apply>
                        <minus/>
                        <ci>b</ci>
                    </apply>
                </apply>
                <ci>d</ci>
            </apply>
        </apply>
    </apply>
</math>
