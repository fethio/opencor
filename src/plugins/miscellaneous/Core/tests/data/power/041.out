<math xmlns="http://www.w3.org/1998/Math/MathML">
    <mrow>
        <mi mathvariant="italic">a</mi>
        <mo>=</mo>
        <mo>−</mo>
        <msup>
            <mroot>
                <mrow>
                    <mo>−</mo>
                    <mi mathvariant="italic">b</mi>
                </mrow>
                <mrow>
                    <mi mathvariant="italic">c</mi>
                </mrow>
            </mroot>
            <mi mathvariant="italic">d</mi>
        </msup>
    </mrow>
</math>
